cmd/kubeadm/app/constants/constants.go

```diff
@@ -417,6 +417,9 @@ const (
 
 	// PauseVersion indicates the default pause image version for kubeadm
 	PauseVersion = "3.4.1"
+
+	// SiteLocalRange is ipv6 site-local range which is deprecated.
+	SiteLocalRange = "FEC0::/10"
```

Comment:
i think we can skip adding this to constants.go

Reply:
Updated